app/Infrastructure/Console/Kernel.php

```diff
@@ -27,14 +38,15 @@ protected function schedule(Schedule $schedule): void
         $schedule->job(CheckYearPeriod::class)
             ->yearlyOn(1, 1, "01:00");
 
-        $this->scheduleDatabaseBackup($schedule);
-    }
+        $schedule->command("cache:prune-stale-tags")->hourly();
```

Comment:
I would like to use class name instead of raw string command signature.

Reply:
Changed.